packages/deploy/src/utils/deploy.ts

```diff
@@ -0,0 +1,29 @@
+import { Artifact, ContractArtifact, DeployContractRequest, RequestArtifact } from '../models';
+
+export function reduceArtifactSize(req: RequestArtifact): Artifact | undefined {
+  const { artifactPayload: artifact, contractName: name, contractPath: path } = req;
+  if (!artifact) return undefined;
+
+  const artifactObj = JSON.parse(artifact);
+
+  const contract: ContractArtifact | undefined = artifactObj.output.contracts[path]?.[name];
+  if (!contract) throw new Error(`Contract ${name} not found in artifact ${artifact}`);
+
+  return {
```

Comment:
Looks like we are omitting the `input` we need to include everything in the input as it is used for etherscan verification

Reply:
Did this pass when you tested it? It should have failed without an input

Reply:
Interesting, I would have expected this error https://github.com/OpenZeppelin/defender/blob/da6122243a9193d795c1c79912876e8bd178e322/api/src/managers/deployment-manager.ts#L660

Reply:
Weird, I would have expected it to fail in `parseArtifactInput`

Reply:
I think it's because we call that in a deferred execution `deployment-status-iterator`, so the deployment works but the iterator fails

Reply:
Thanks! Will fix that